project/plugins.sbt

```
@@ -1,3 +1,4 @@
-addSbtPlugin("org.scalameta"  % "sbt-scalafmt"   % "2.4.6")
-addSbtPlugin("ch.epfl.scala"  % "sbt-scalafix"   % "0.10.0")
-addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.10")
+addSbtPlugin("org.scalameta"             % "sbt-scalafmt"   % "2.4.6")
+addSbtPlugin("ch.epfl.scala"             % "sbt-scalafix"   % "0.10.0")
+addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat"   % "0.2.3")
```

```suggestion
addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat"   % "0.3.0")
```

Just released :trollface: